### DISCUSSION TITLE
NextJs 14 Build Locally and push to github then just start the server.

### DISCUSSION THREAD

**Discussion by clevermiraz** [vercel/next.js#63267] [Help]
### Summary

I have a small app made with nextjs14, 
when I try to build the app on the server for low memory build process is crushed.

so, now i want to build that app locally and on server just start the command `npm run start`

is it is possible?

by the way, the build size is so big. 

what should I do? what is a good approach?

**Comment by icyJoseph** [comment]
Hi,

Have you looked into doing `standalone` builds? 

- https://nextjs.org/docs/app/api-reference/next-config-js/output#automatically-copying-traced-files

That might just be the thing to help out with the low memory.

**Reply by clevermiraz** [reply]
Great But I Want Just push `.next` folder and later on just start the server
